logstash-core/lib/logstash/pipeline.rb

```diff
@@ -68,9 +71,16 @@ def non_reloadable_plugins
 
   private
 
+  def added_source_line_column(pipeline_config, plugins)
+    for plugin in plugins do
```

Comment:
nitpick: use `Enumerable#each`
```suggestion
    plugins.each do |plugin|
```